pkg/controllers/cluster/azure.go

```diff
@@ -0,0 +1,48 @@
+package cluster
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	azurev1 "sigs.k8s.io/cluster-api-provider-azure/api/v1beta1"
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+
+	configv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/cluster-capi-operator/pkg/controllers"
+)
+
+func (r *ClusterReconciler) reconcileAzureCluster(ctx context.Context, azurePlatformStatus *configv1.AzurePlatformStatus) error {
+	if azurePlatformStatus == nil {
+		return errors.New("AzurePlatformStatus can't be nil")
+	}
+
+	azureCluster := &azurev1.AzureCluster{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      r.clusterName,
+			Namespace: controllers.DefaultManagedNamespace,
+			Annotations: map[string]string{
+				clusterv1.ManagedByAnnotation: "",
```

Comment:
Remind me where we landed on having a value for this annotation?

Reply:
I don't think we discussed it

Reply:
I would have expected the owner to put a value in here no? So that you can see from the "api" who to talk to when it breaks, or so that you can tell that one specific operator is owning it and not another